am-ui/nginx.conf

```diff
@@ -125,6 +125,18 @@ http {
 			# proxy_cache_bypass $http_upgrade;
 		}
 
+		location /users {
+			proxy_pass https://marketplace-service/users;
```

Comment:
If we start the frontend as a developer, the NGINX is not used, because the role of the reverse proxy is done by angular proxy configuration. However, if the application is started only locally using docker, the NGINX is used.